src/test/java/com/team4/testingsystem/controllers/TestsControllerTest.java

```diff
@@ -107,65 +111,89 @@ void startNotAssignedSuccess() {
 
-        testsController.startAssigned(1L);
+        testsController.startAssigned(GOOD_TEST_ID);
 
-        verify(testsService).start(1L);
+        verify(testsService).start(GOOD_TEST_ID);
     }
 
     @org.junit.jupiter.api.Test
     void startAssignedFail() {
 
-        doThrow(TestNotFoundException.class).when(testsService).start(42L);
+        doThrow(TestNotFoundException.class).when(testsService).start(BAD_TEST_ID);
 
-        Assertions.assertThrows(TestNotFoundException.class, () -> testsController.startAssigned(42L));
+        Assertions.assertThrows(TestNotFoundException.class, () -> testsController.startAssigned(BAD_TEST_ID));
     }
 
     @org.junit.jupiter.api.Test
     void finishSuccess() {
 
-        testsController.finish(1L, 1);
+        testsController.finish(GOOD_TEST_ID, 1);
 
-        verify(testsService).finish(1L, 1);
+        verify(testsService).finish(GOOD_TEST_ID, 1);
     }
 
     @org.junit.jupiter.api.Test
     void finishFail() {
 
-        doThrow(TestNotFoundException.class).when(testsService).finish(42L, 42);
+        doThrow(TestNotFoundException.class).when(testsService).finish(BAD_TEST_ID, 42);
 
-        Assertions.assertThrows(TestNotFoundException.class, () -> testsController.finish(42L, 42));
+        Assertions.assertThrows(TestNotFoundException.class,
+                () -> testsController.finish(BAD_TEST_ID, 42));
     }
 
     @org.junit.jupiter.api.Test
     void updateEvaluationSuccess() {
 
-        testsController.updateEvaluation(1L, 1);
+        testsController.updateEvaluation(GOOD_TEST_ID, 1);
 
-        verify(testsService).updateEvaluation(1L, 1);
+        verify(testsService).updateEvaluation(GOOD_TEST_ID, 1);
     }
 
     @org.junit.jupiter.api.Test
     void updateEvaluationFail() {
 
-        doThrow(TestNotFoundException.class).when(testsService).updateEvaluation(42L, 42);
+        doThrow(TestNotFoundException.class).when(testsService).updateEvaluation(BAD_TEST_ID, 42);
 
-        Assertions.assertThrows(TestNotFoundException.class, () -> testsController.updateEvaluation(42L, 42));
+        Assertions.assertThrows(TestNotFoundException.class,
+                () -> testsController.updateEvaluation(BAD_TEST_ID, 42));
     }
 
+
     @org.junit.jupiter.api.Test
-    void removeByIdSuccess() {
+    void assignCoachSuccess() {
+        testsController.assignCoach(GOOD_TEST_ID, GOOD_USER_ID);
 
-        testsController.removeById(1L);
+        verify(testsService).assignCoach(GOOD_TEST_ID, GOOD_USER_ID);
+    }
 
-        verify(testsService).removeById(1L);
+    @org.junit.jupiter.api.Test
+    void assignCoachFailFirst() {
+        doThrow(UserNotFoundException.class).when(testsService).assignCoach(GOOD_TEST_ID, BAD_USER_ID);
+
+        Assertions.assertThrows(UserNotFoundException.class,
+                () -> testsController.assignCoach(GOOD_TEST_ID, BAD_USER_ID));
     }
 
     @org.junit.jupiter.api.Test
-    void removeByIdFail() {
+    void assignCoachFailSecond() {
```

Comment:
Let's name the tests with more sense, than just `First`, `Second`.
Maybe `UserNotFound` and `TestNotFound`